core/domain/topic_services.py

```diff
@@ -570,6 +571,13 @@ def update_topic_and_subtopic_pages(
                 subtopic_page_change_list)
     create_topic_summary(topic_id)
 
+    # TODO(DubeySandeep): Remove this import from here and create a
```

Comment:
You should be doing topic_fetchers in this PR, or a PR before it.

Reply:
Do another PR before it (with story fetchers as well). Merge it in and then have this PR depend on it.